Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
+    filter(Gender == "FEMALE") %>% 
+    set_rownames(levels(df.out$country)) %>% 
+    dplyr::select(-Gender, -Country)
+
+# Labor supply choice tables
+df.female = df.out %>% filter(gender == "FEMALE")
+vars      = names(df.out)[grep("labor", names(df.out))][3:1] # invert
+
+labor.supply.f    = lapply(vars, function(x) group.share(df.female, x, 1))
+labor.supply.f.df = do.call(cbind.data.frame, labor.supply.f) %>% 
+    set_colnames(c("Nonparticipation", "Half time", "Full time"))
+
+
+```
+
+
+
+
+## Empirical Results
+
+```{r, eval = TRUE, results = 'asis', warning = FALSE, comment = FALSE, message = FALSE, echo = FALSE}
+
+library("knitr")
+knitr::kable(t(round(sum.stats.tot, 2)), caption = "Summary Statistics over all groups")
+
+
+knitr::kable(labor.part.m, caption = "Men: Labor Force Participation Rate by age")
+knitr::kable(labor.part.f, caption = "Women: Labor Force Participation Rate by age")
+
+```
+
+
+Having obtained the share of labor force participation grouped by gender, country and age as well as the according health variables, striking values can be presented.
+
+Among the selected countries, France, Greece and Belgium have the fewest people among the oldest age group, which is ranging from 60 to 64 years. In the middle age group, ranging from 54-59, the German population is sparsely represented. Within the youngest age group, including people from 50 to 54 years old, Italy and Austria have the lowest share.  
+Instead, Austria represents by far the largest share in the oldest age group. Belgium, Italy, the Netherlands and Sweden dominate the highest ranks in the group ranging from 54 to 59 years. Additionally, Greece solely leads the share in the youngest age group. 
+
+Concerning health statistics, Switzerland and the Scandinavian countries stand out positively. Together, they from a group that has an overall low percentage of obese people while achieving high shares of self-reported good health. Among all other countries, the Swiss have the least number of chronic diseases and fewest overweight. Only the Greek have lower share of ADL's. 
+
+Switzerland and the Nordic countries additionally present the least non-participation rates and the highest full-time working rates. 
+Summary Statistics show that those countries, which strive in health topics are related to high labor force participation rates in all categories. Despite this, the only candidate for multicollinearity is the age dummy in the age group 50, which is omitted before model calculation. 
+
+Generally, labor force participation is higher for males than for females. This applies to part-time work, too.
+
+The overall results of the descriptive statistics align well with @kalwij2005labour
+
+\newpage
+
 
 
 \section{Crosssectional probit regression}
-JULIAN
-\subsection{Theory and Design}
-\subsection{Implementation}
-\subsection{Empirical Results}
+
+## Theory and Design 
+
+Probit models allow for the estimation of binary, random response variables.  
```

Comment:
Please find a source for that (e.g. Woolridge). Yu should also mention that it is a popular classification model and often estimated with ML. You dont need to mention Wald yet

Reply:
sources are still on my computer. uploading this evening.